ibis/backends/pandas/__init__.py

```diff
@@ -298,6 +298,10 @@ def to_pyarrow(
         output = pa.Table.from_pandas(
             self.execute(table_expr, params=params, limit=limit, **kwargs)
         )
+
+        output = output.drop_columns(
+            filter(lambda col: col.startswith("__index_level_"), output.column_names)
```

Comment:
Can you add a short comment about the trailing `_` in `__index_level_`? I almost reviewed it as a mistake :)